tests/test_supply_curve_exclusions.py

```diff
@@ -248,6 +248,32 @@ def test_no_excl(ds_slice):
         assert np.allclose(truth, test)
 
 
+def test_multiple_excl_fractions():
+    """
+    Test that multiple fraction exclusions are handled properly
+    """
+    excl_h5 = os.path.join(TESTDATADIR, 'ri_exclusions', 'ri_exclusions.h5')
+
+    excl_dict = {'ri_padus': {'exclude_values': [1, ], 'weight': 0.5,
+                              'exclude_nodata': True}}
+    with ExclusionMaskFromDict(excl_h5, layers_dict=excl_dict) as f:
+        truth = f.mask
+
+    excl_dict = {'ri_padus': {'exclude_values': [1, ], 'weight': 0.25,
+                              'exclude_nodata': True}}
+    with ExclusionMaskFromDict(excl_h5, layers_dict=excl_dict) as f:
+        truth = np.minimum(truth, f.mask)
+
+    excl_dict = {'ri_padus': {'exclude_values': [1, ], 'weight': 0.25,
+                              'exclude_nodata': True},
+                 'ri_padus': {'exclude_values': [1, ], 'weight': 0.25,
+                              'exclude_nodata': True}}
+    with ExclusionMaskFromDict(excl_h5, layers_dict=excl_dict) as f:
+        test = f.mask
+
+    assert np.allclose(test, truth)
```

Comment:
Would you also want to add a line checking that the minimum value greater than zero is not less than 0.25? Simple check that would be worth doing, imo.

Reply:
@grantbuster we'd potentially have percent inclusions that are less than 25%, though. Could be any number between 0 and 1, really.

Reply:
@nickwg03, this is a pytest. This would be just a verification on the exclusion data based on the test inputs. I'm not implying anything about production outputs.

Reply:
Haha, okay. I'll stay in my lane.

Reply:
Lol sorry if that was blunt, I appreciate you reviewing our PR's :)